### ISSUE TITLE
"Mumble Integration" DMCA Takedown Notice

### ISSUE THREAD

**Issue by magneticflux-** [GeheimagentNr1/Issues#1]
I am the copyright holder of the code in these repositories:
- https://github.com/magneticflux-/fabric-mumblelink-mod
- https://github.com/magneticflux-/java-mumble-link
- https://github.com/magneticflux-/jvm-shared-memory

It has come to my attention that your mod "Mumble Integration", published on CurseForge [here](https://www.curseforge.com/minecraft/mc-mods/mumble-integration) includes substantial code that is under copyright and licensed under the LGPLv3. By distributing that code without credit and without source code, you are in violation of that license.

Please remove all offending content immediately, or I will proceed to file an official DMCA complaint.

**Comment by GeheimagentNr1** [comment]
I can check this at the 05.01.2021 at the earliest.
I check it as soon as possible. Furthermore, I achieved the mod files on CurseForge so there not downloadable.

**Comment by magneticflux-** [comment]
It is not currently under a license, so all rights are reserved.

I had planned to license it under LGPLv3 as I did in https://github.com/magneticflux-/jvm-shared-memory/commit/3dc13f6cd89424a5efe32d251bb03d05f0b9afa3 but never got around to it. I will license it under the LGPLv3.

**Comment by GeheimagentNr1** [comment]
If I'm correct LGPLv3 licence says, that the source code must be open source and the projects build with it have to have at least the LGPLv3 licence too. Am I correct? Have I forgotten something?

When this is correct, I will fix these issues and send you an information in this thread with repo link and a jar version of the project, so you can check, that everything is correct then.

**Comment by magneticflux-** [comment]
This is the text of the license: https://www.gnu.org/licenses/lgpl-3.0.html
For your convenience, there is also an unofficial German translation here: http://www.gnu.de/documents/lgpl-3.0.de.html

The code I wrote represents a "Library".
The code you published represents a "Combined Work", which section 4 covers. There are 5 requirements of such a work listed. The LGPL license does not require that a Combined Work be also licensed under the LGPL, only that a Combined Work:

1. "Give prominent notice [...] that the Library is used in it"
2. "Accompany the Combined Work with a copy of the GNU GPL and this license document."
3. "[...] include the copyright notice for the Library among these notices [...]"
4. Allow the a user some mechanism to modify the Library being used by the Combined Work.
5. Provide Installation Information for a user to use the Combined Work with a modified version of the Library.

Quotations are taken from the LGPLv3, the rest are my own paraphrases and not legal advice.

**Comment by GeheimagentNr1** [comment]
I hope it is ok, that I only archived the files on CurseForge and will fix the licencing in the new year.
And I hope, that it is ok, that I will let you review, that I didn't do anything wrong with the licencing, when I have done it.

**Comment by GeheimagentNr1** [comment]
I have created a new repository, where I have added a notice that the library is used, that you are the copyright holder and in the lib_licences folder are copies of the gpl-3.0 and lgpl-3.0 licences. And I added an explanation how to change the version of the library.
Here is the link: https://github.com/GeheimagentNr1/MumbleIntegration
I didn't add the source and issues link to the CurseForge page yet, because I what to wait until you verified that everything is correct now. Before publishing the new version I with add the links to the source code and the issue page of the repository.
I attached the new version jar file to this post, if you want to view them:
[jar_files.zip](https://github.com/GeheimagentNr1/Issues/files/5762417/jar_files.zip)
